src/config.rs

```diff
@@ -457,23 +457,22 @@ mod tests {
         assert_eq!(projects_count, 0);
     }
 
-    #[test]
-    fn test_maybe_expand_home_dir() {
-        let expected = Ok(String::from("/home/vardy/tod.cfg"));
-        let actual = maybe_expand_home_dir(expected.clone().unwrap());
-        assert_eq!(expected, actual);
-
-        let actual = maybe_expand_home_dir("~/tod.cfg".to_string());
-
-        let split = actual.unwrap();
-        let mut split = split.split('/');
-
-        assert_eq!(split.next(), Some(""));
-        assert_eq!(split.next(), Some("home"));
-        // This is machine dependent
-        split.next();
-        assert_eq!(split.next(), Some("tod.cfg"));
-    }
+    // #[test]
+    // fn test_maybe_expand_home_dir() {
+    //     let expected = Ok(String::from("~/tod.cfg")); // This is machine/user dependent
+    //     let actual = maybe_expand_home_dir(expected.clone().unwrap());
+    //     assert_eq!(expected, actual);
+
+    //     let actual = maybe_expand_home_dir("/Users/tod.cfg".to_string());
+
+    //     let split = actual.unwrap();
+    //     let mut split = split.split('/');
+
+    //     assert_eq!(split.next(), Some(""));
+    //     assert_eq!(split.next(), Some("home"));
+    //     split.next();
+    //     assert_eq!(split.next(), Some("tod.cfg"));
+    // }
```

Comment:
Created an issue for you to resolve later, so it doesn't block current PR: https://github.com/alanvardy/tod/issues/809

Reply:
Thx - will update this, need to create separate tests based on platform, also will probably rewrite function to use dirs since that is already present, no need for homedir crate.